tests/cstor/volume/suite_test.go

```diff
@@ -37,21 +38,23 @@ var (
 	openebsNamespace      = "openebs"
 	nsName                = "cstor-provision"
 	scName                = "cstor-volume"
-	openebsCASConfigValue = "- name: ReplicaCount\n  value: 1\n- name: StoragePoolClaim\n  value: sparse-pool-auto"
-	openebsProvisioner    = "openebs.io/provisioner-iscsi"
-	spcName               = "sparse-pool-auto"
-	nsObj                 *corev1.Namespace
-	scObj                 *storagev1.StorageClass
-	spcObj                *apis.StoragePoolClaim
-	pvcObj                *corev1.PersistentVolumeClaim
-	spcList               *apis.StoragePoolClaimList
-	targetLabel           = "openebs.io/target=cstor-target"
-	accessModes           = []corev1.PersistentVolumeAccessMode{corev1.ReadWriteOnce}
-	capacity              = "5G"
-	annotations           = map[string]string{
-		string(apis.CASTypeKey):   string(apis.CstorVolume),
-		string(apis.CASConfigKey): openebsCASConfigValue,
-	}
+	openebsCASConfigValue = `
+- name: ReplicaCount
+  value: count
```

Comment:
can we have `count` as `${count}` or `$count`

Reply:
I just want to represent it as  a proper placeholder.

Reply:
done